certbot-nginx/tests/testdata/etc_nginx/invalid_unicode_comments.conf

```diff
@@ -0,0 +1,7 @@
+# This configuration file is saved with EUC-KR (a.k.a. cp949) encoding,
+# including some Korean alphabets.
```

Comment:
I guess it's now only one Korean alphabet (Hangul). :slightly_smiling_face:

Reply:
My intention was that, the file includes some Korean letters when read with correct encoding. I clerified it in ddf68aea8039b81940b7caf731405ef10613b5db.